content/blog/2024-11-18-yace-joining-prometheus-community.md

```diff
@@ -0,0 +1,26 @@
+---
+title: YACE is joining Prometheus Community
+created_at: 2024-11-18
+kind: article
+author_name: Thomas Peitz (@thomaspeitz)
+draft: true
+---
+
+Yet Another Cloudwatch Exporter (YACE) has officially joined the Prometheus community! This move will make it more accessible to users and open new opportunities for contributors to enhance and maintain the project.
+
+## The early days
```

Comment:
Do you mind adding a quick section on how you can use it if you are Prometheus and Cloudwatch users but never seen this exporter? Would be useful to use this blog post opportunity to get new users here! 🤗

Reply:
Love the idea. Will add this in the evening :)

Reply:
Related, we should probably write a migration guide for the existing cloudwatch_exporter. (There's some idea that we'll deprecate it in favor of YACE).

Maybe we can rename YACE -> cloudwatch_exporter eventually. :grin:

Reply:
Would be great to write down a migration guide.

IMO let's add the section that @bwplotka proposed and aim to get this published today, then we can followup with other content later on.

Reply:
Added as last paragraph